### PR TITLE
Change homepage hero

### PR DESCRIPTION
**Pull request by gabrielpoca**
Why:

This change is part of the website redesign. We are moving towards a
simpler version. I started by updating the hero.

I also updated the scss_lint gem because my editor was asking and I didn't see a problem there.

This change addresses the need by:

* Updating the homepage's hero.
* Removing tons of code.

### CODE REVIEW

**Comment by gabrielpoca** [comment]
@naps62 could you look into this?

File: source/javascripts/all.js [hunk 1]
```diff
@@ -8,9 +8,6 @@
 //= require _input
 //= require _animation_helpers
 //= require _custom_events
-//= require homepage/_lines
-//= require homepage/_planets
-//= require homepage/_hero_parallax
```
Comment (naps62) [inline source/javascripts/all.js]: weren't these being used on the Work page hero as well?

Comment (gabrielpoca) [inline source/javascripts/all.js]: you're right. I added them back

**Comment by gabrielpoca** [comment]
@naps62 bump

**Comment by naps62** [comment]
:+1:

**Comment by gabrielpoca** [comment]
I will close this PR, but I won't merge the code yet. I intend to make pull requests to this branch from now on, until the full homepage redesign is complete and I can merge the code to master.

**Pull request closed without merging**